### PR TITLE
[synthetics] Add note on required privileges for enabling monitor management

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Adds a note on required privileges for monitor management. I'm not sure about the phrasing -- suggestions are very welcome!

### CODE REVIEW

**Review by paulb-elastic — APPROVED** [review]
The location and context of the update LGTM.  I'll defer to @dominiqueclarke's review of the details.

File: docs/en/observability/uptime-set-up-app.asciidoc [hunk 1]
```diff
@@ -10,6 +10,15 @@ To try this beta functionality, first you need to enable Monitor Management:
 . Click **Monitor Management**.
 . Review the terms and conditions and click **Enable** to start using Monitor Management.
 
+[IMPORTANT]
+======
+To enable Monitor Management, you must be logged into {kib} as a user with
+the following privileges:
+
+* {ref}/security-privileges.html#privileges-list-cluster[Cluster privileges]: `monitor`, `read_ilm`, `read_pipeline`, and at least one of `manage_own_api_key`, `manage_security`, or `manage_api_key`
+* {ref}/security-privileges.html#privileges-list-indices[Indices privileges]: `view_index_metadata`, `create_doc`, `auto_configure`
```
Comment (dominiqueclarke) [inline docs/en/observability/uptime-set-up-app.asciidoc:19]: We should note that these indices privileges must be for `synthetics-*`

Comment (colleenmcginnis) [reply docs/en/observability/uptime-set-up-app.asciidoc:19]:
```suggestion
* {ref}/security-privileges.html#privileges-list-indices[Indices privileges] for `synthetics-*`: `view_index_metadata`, `create_doc`, `auto_configure`
```

**Review by dominiqueclarke — APPROVED** [review]
LGTM

**Pull request merged**